test/cmdlineTests/standard_output_selection_ethdebug_optimize_iroptimized/input.json

```diff
@@ -0,0 +1,24 @@
+{
+ "language": "Solidity",
```

Comment:
Come on, pass it through `jq` at least :) 1-space indents are just WTF.

Reply:
I totally agree here - not sure what happened 😅